scripts/ci_aws_bootstrap.sh

```diff
@@ -2,76 +2,90 @@
+    exit 0
+}
 
 upload_pipeline() {
     echo "Uploading pipeline..."
     ls .buildkite || buildkite-agent annotate --style error 'Please merge upstream main branch for buildkite CI'
+    
+    # Install minijinja-cli
     curl -sSfL https://github.com/mitsuhiko/minijinja/releases/latest/download/minijinja-cli-installer.sh | sh
     source /var/lib/buildkite-agent/.cargo/env
+    
     if [ $BUILDKITE_PIPELINE_SLUG == "fastcheck" ]; then
-        if [ ! -e ".buildkite/test-template-fastcheck.j2" ]; then
-            curl -o .buildkite/test-template-fastcheck.j2 https://raw.githubusercontent.com/vllm-project/buildkite-ci/main/scripts/test-template-fastcheck.j2
-        fi
-        cd .buildkite && minijinja-cli test-template-fastcheck.j2 test-pipeline.yaml > pipeline.yml
-        cat pipeline.yml
-        buildkite-agent pipeline upload pipeline.yml
-        exit 0
+        handle_fastcheck
+    else
+        handle_regular_pipeline
     fi
-    if [ ! -e ".buildkite/test-template.j2" ]; then
+}
+
+handle_fastcheck() {
+    [ ! -e ".buildkite/test-template-fastcheck.j2" ] && \
+        curl -o .buildkite/test-template-fastcheck.j2 https://raw.githubusercontent.com/vllm-project/buildkite-ci/main/scripts/test-template-fastcheck.j2
+    
+    cd .buildkite && minijinja-cli test-template-fastcheck.j2 test-pipeline.yaml > pipeline.yml
+    cat pipeline.yml
+    buildkite-agent pipeline upload pipeline.yml
+    exit 0
+}
+
+handle_regular_pipeline() {
+    [ ! -e ".buildkite/test-template.j2" ] && \
         curl -o .buildkite/test-template.j2 https://raw.githubusercontent.com/vllm-project/buildkite-ci/main/scripts/test-template-aws.j2
-    fi
+    
     if [ -e ".buildkite/pipeline_generator/pipeline_generator.py" ]; then
         python -m pip install click pydantic
         python .buildkite/pipeline_generator/pipeline_generator.py --run_all=$RUN_ALL --list_file_diff="$LIST_FILE_DIFF"
         buildkite-agent pipeline upload .buildkite/pipeline.yaml
-        exit 0
+    else
+        cd .buildkite
+        echo "List file diff: $LIST_FILE_DIFF"
+        echo "Run all: $RUN_ALL"
+        minijinja-cli test-template.j2 test-pipeline.yaml -D list_file_diff="$LIST_FILE_DIFF" -D run_all="$RUN_ALL" > pipeline.yml
+        buildkite-agent pipeline upload pipeline.yml
     fi
-    cd .buildkite
-    echo "List file diff: $LIST_FILE_DIFF"
-    echo "Run all: $RUN_ALL"
-    minijinja-cli test-template.j2 test-pipeline.yaml -D list_file_diff="$LIST_FILE_DIFF" -D run_all="$RUN_ALL" > pipeline.yml
-    buildkite-agent pipeline upload pipeline.yml
     exit 0
 }
 
 get_diff() {
-    $(git add .)
-    echo $(git diff --name-only --diff-filter=ACMDR $(git merge-base origin/main HEAD))
+    git add .
+    git diff --name-only --diff-filter=ACMDR $(git merge-base origin/main HEAD)
 }
 
 get_diff_main() {
-    $(git add .)
-    echo $(git diff --name-only --diff-filter=ACMDR HEAD~1)
+    git add .
+    git diff --name-only --diff-filter=ACMDR HEAD~1
```

Comment:
this is `git show --pretty="" --name-only` ?

and why `ACMDR` is required? do you know?